app/src/lib/checks/index.js

```diff
@@ -152,12 +152,11 @@ function check_for_unbalanced_brackets(checked_tokens) {
 	 * @returns {CountTracker}
 	 */
 	function sum(tracker, {token}) {
-		const open_count = token.match(/\[/g)?.length ?? 0
-		const close_count = token.match(/\]/g)?.length ?? 0
-
-		tracker.open += open_count
-		tracker.close += close_count
-
+		if (token === '[') {
```

Comment:
not a blocker here but I'd like to strive to avoid conditionals if at all possible.  I'd be curious to understand this change a little more.  Thanks!